src/models/clients.js

```diff
@@ -22,6 +22,9 @@ module.exports = (sequelize, DataTypes) => {
       defaultValue: () => {
         return randomBytes(64).toString('hex');
       }
+    },
+    redirectUrl: {
+      type: DataTypes.STRING(2048)
```

Comment:
`allowNull: false` makes the field mandatory. I was thinking that we should probably allow creating new clients without auth redirection urls. Some of these clients might not want to use the APIs that will require user authentication (I think for now we will have only an API to bookmark sensors requiring user authentication). Some clients might only be interested on pushing sensors data to the server. And for sensors authentication we can't use a redirection url (we can't redirect to a sensor) so the authentication process will be different.

Allowing to configure several redirection urls sounds good.